source/aggregation/agg-exp-ops.txt

```diff
@@ -0,0 +1,1183 @@
+       | `ofDateArray() <{+core-api+}/com/mongodb/client/model/mql/MqlValues.html#ofDateArray(java.time.Instant...)>`__
+       | `ofIntegerArray() <{+core-api+}/com/mongodb/client/model/mql/MqlValues.html#ofIntegerArray(int...)>`__
+       | `ofNumberArray() <{+core-api+}/com/mongodb/client/model/mql/MqlValues.html#ofNumberArray(double...)>`__
+       | `ofStringArray() <{+core-api+}/com/mongodb/client/model/mql/MqlValues.html#ofStringArray(java.lang.String...)>`__
+
+     - Returns an array of ``MqlValue`` types corresponding to the provided array of primitives.
+
+   * - `ofEntry() <{+core-api+}/com/mongodb/client/model/mql/MqlValues.html#ofEntry(com.mongodb.client.model.mql.MqlString,T)>`__
+     - Returns an entry value.
+
+   * - `ofMap() <{+core-api+}/com/mongodb/client/model/mql/MqlValues.html#ofMap()>`__
+     - Returns an empty map value.
+
+   * - `ofNull() <{+core-api+}/com/mongodb/client/model/mql/MqlValues.html#ofNull()>`__
+     - Returns the null value as exists in the Query API.
+
+.. important::
+
+   When you provide a value to one of these methods, the driver treats
+   it literally. For example, ``of("$x")`` represents the string value
+   ``"$x"``, rather than a field named ``x``.
+
+Refer to any of the sections under :ref:`Operations
+<kotlin-sync-aggregation-expression-ops-section>` for examples using these
+methods.
+
+.. _kotlin-sync-aggregation-expression-ops-section:
+
+Operations
+----------
+
+The following sections provide information and examples for
+aggregation expression operations available in the driver.
+The operations are categorized by purpose and functionality.
+
+Each section has a table that describes aggregation methods
+available in the driver and corresponding expression operators in the
+Query API. The method names link to API documentation and the
+aggregation pipeline operator names link to descriptions and examples in
+the Server manual documentation. While each method is effectively
+equivalent to the corresponding Query API expression, they may differ in
+expected parameters and implementation.
+
+.. note::
+
+   The driver generates a Query API expression that may be different
+   from the Query API expression provided in each example. However,
+   both expressions will produce the same aggregation result.
+
+.. important::
+
+   The driver does not provide methods for all aggregation pipeline operators in
+   the Query API. To use an unsupported operation in an
+   aggregation, you must define the entire expression using the BSON ``Document``
+   type.
+   
+.. TODO add to note To learn more about the ``Document`` type, see :ref:`<kotlin-sync-document-format>`.
+
+Arithmetic Operations
+~~~~~~~~~~~~~~~~~~~~~
+
+You can perform an arithmetic operation on a value of type ``MqlInteger`` or
+``MqlNumber`` using the methods described in this section.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
```

Comment:
Similar to above, and for all the Operation subsections, do we need the full list of methods here, or can we just link to the API documentation and keep the subsections dedicated to examples?

Reply:
Since the tables also provide the 1:1 correspondence with the Query API operators, I think that the tables are valuable